### ISSUE TITLE
No Max set causes repeat ingame warnings

### ISSUE THREAD

**Issue by ghostorchidgaming** [mattsemar/dsp-personal-logistics#2]
Great plugin so far!

If you only set a Min and no Max, you will repeatedly get warnings about not having enough energy to request item.
I just figured that not having a Max set meant that the item would only be resupplied when it goes below the Min.  Are we forced to set a Max as well?

Thanks!

**Comment by mattsemar** [comment]
I think I need to do some work around this. The original idea I had was that if max is set then extra items will get sent to logistics network. With no max then you can have as many as you like in your inventory and they will just be ignored.

I think it's confusing though, and needs some more thought as to how it should work.

**Comment by mattsemar** [comment]
Ok, I think I got this sorted. There were two unrelated things happening here. First was that if the station that your requested items happened to be pulled from did not have enough power to send the items to you the mod would take 10% of your mecha's energy until the needed amount of energy was supplied. I changed that so that, if needed, up to 90% of your mecha energy would be used in bringing items in. I also a check to limit spamming of messages shown in this case.
